syncd/FlexCounter.cpp

```diff
@@ -511,8 +519,72 @@ class CounterContext : public BaseCounterContext
         }
     }
 
+    void addVoqEntryToChassisAppDb(_In_ const sai_object_id_t rid,
```

Comment:
first parameter should be also in separate line

Reply:
Done.